### DISCUSSION TITLE
Github search with workflow files?

### DISCUSSION THREAD

**Discussion by kaihendry** [community/community#25461] [Actions]
<p>How does one use Github search to filter by github/workflows files?</p>
<p>For example with Travis CI, if I wanted to learn about CI practices in Go projects I would perform something like:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/search?q=%22language:%2Bgo%22%2Bfilename:.travis.yml" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/765871/181091220-2dda53ba-89c3-429f-a6c4-2d4c002464b8.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://github.com/search?q=%22language:%2Bgo%22%2Bfilename:.travis.yml" target="_blank" rel="noopener nofollow ugc">Build software better, together</a></h3>

<p>GitHub is where people build software. More than 50 million people use GitHub to discover, fork, and contribute to over 100 million projects.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/kaihendry">@kaihendry</a>,</p>
<p>I think you can use the similar methods to search workflow files. The main difference is that the workflow files normally are in the “<strong>.github/workflows</strong>” directory. More details about GitHub search, you can see “<a href="https://docs.github.com/en/free-pro-team@latest/github/searching-for-information-on-github/getting-started-with-searching-on-github" rel="noopener nofollow ugc"><strong>Getting started with searching on GitHub</strong></a>”.</p>
<p>In addition, the <a href="https://github.community/c/github-actions/41"><strong>GitHub Actions board</strong></a> of GitHub Community is used for the discussions and questions about <a href="https://docs.github.com/en/free-pro-team@latest/actions" rel="noopener nofollow ugc"><strong>GitHub Actions</strong></a>. If you need help with anything related to workflow configuration, such as syntax, GitHub-hosted runners, or building actions, you can look for an existing topic or start a new one in this board.</p>
<p>About your question, I recommend that you can report it in the <a href="https://github.community/c/how-to-use-git-and-github/21"><strong>How to use Git and GitHub board</strong></a>.</p>

**Comment by kaihendry** [comment]
<p>Report it, as in Repost it in another forum?</p>
<p>My question is about utilizing and learning workflows. What a shame.</p>

**Comment by Simran-B** [comment, marked as answer]
<p>Here’s a search query that finds workflow files that contain the term “go”:</p>
<p><a href="https://github.com/search?l=YAML&amp;q=go%20path%3A.github%2Fworkflows&amp;type=Code" rel="noopener nofollow ugc">https://github.com/search?l=YAML&amp;q=go%20path%3A.github%2Fworkflows&amp;type=Code</a></p>
